source/extensions/tracers/datadog/time_util.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+/**
+ * This file contains functions related to time points and durations.
+ *
+ * Envoy has a time type that contains both a system time point and a steady
```

Comment:
nit: Envoy has a TimeSource abstraction that provides the system time point and a steady ....


The time types themselves just have one of those.

Reply:
Yeah, I noticed that last week! Glad you caught it, I'll change the wording to be accurate.